core/Cargo.toml

```diff
@@ -14,9 +14,8 @@ name = "limbo_core"
 path = "lib.rs"
 
 [features]
-default = ["fs", "json", "uuid", "time"]
+default = ["fs", "uuid", "time"]
 fs = ["limbo_ext/vfs"]
-json = ["dep:jsonb", "dep:pest", "dep:pest_derive", "dep:serde", "dep:indexmap"]
```

Comment:
We do want to keep the feature flag to allow people to build Limbo _without_ JSON support, similar to what SQLite allows with the `SQLITE_ENABLE_JSON` build option.

Reply:
@penberg Yes, my bad, fixed